Assets/Plugins/Source/Editor/Windows/CreatePackageWindow.cs

```diff
@@ -301,7 +301,7 @@ private async void StartCreatePackageAsync(UPMUtility.PackageType type)
                 {
                     // Ternary statement here to prevent a divide by zero problem
                     // ever happening, despite how odd it would be in this case.
-                    float newActualProgress = (0.0f >= value.TotalBytesToCopy)
+                    float newActualProgress = (0.0f <= value.TotalBytesToCopy)
```

Comment:
value.TotalBytesToCopy can be less than or equal to 0. If true, then it's going to try to divide by zero. Was this boolean logic flip intentional?

Reply:
This got fixed via the previous PR being merged into `development`, then `development` being subsequently merged back into this branch.